packages/sample-app/src/sample_chromadb.ts

```diff
@@ -0,0 +1,121 @@
+import * as traceloop from "@traceloop/node-server-sdk";
```

Comment:
We try to make these examples resemble "real-life" scenarios, can you do that here as well? (Similar to the sample app we have in Python)